Kapt always fails with ClassNotFoundException

I'm trying to integrate Showkase in a compose project as a separate Android app module, and I consistently get this error every time I build the project:

```
Caused by: java.lang.ClassNotFoundException: androidx.compose.runtime.Composable
	at com.airbnb.android.showkase.processor.ShowkaseProcessor.init(ShowkaseProcessor.kt:58)
	at org.jetbrains.kotlin.kapt3.base.incremental.IncrementalProcessor.init(incrementalProcessors.kt:41)
	at org.jetbrains.kotlin.kapt3.base.ProcessorWrapper.init(annotationProcessing.kt:175)
	at jdk.compiler/com.sun.tools.javac.processing.JavacProcessingEnvironment$ProcessorState.<init>(JavacProcessingEnvironment.java:686)
	... 41 more
```

I made sure to be using the save versions for `compose`, `compose-activity`, `compose-navigation` and `compose-constraintlayout`. Have tried to disable many other dependencies that might be causing the issue with kapt, but at this point I'm not sure what else to try.

Versions:
```
showkase = "1.0.0-alpha12"
compose = "1.0.0-beta09"
composeConstraintlayout = "1.0.0-alpha07"
composeActivity = "1.3.0-beta02"
composeNavigation = "2.4.0-alpha03"
```

--- a/showkase-processor/build.gradle
+++ b/showkase-processor/build.gradle
@@ -20,14 +20,6 @@ kapt {
     generateStubs = true
 }
 
-sourceSets {
-    main {
-        java {
-            srcDir "${buildDir.absolutePath}/tmp/kapt/main/kotlinGenerated/"
-        }
-    }
-}
-
 java {
     sourceCompatibility = JavaVersion.VERSION_11
     targetCompatibility = JavaVersion.VERSION_11

--- a/showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt
+++ b/showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt
@@ -55,10 +55,10 @@ class ShowkaseProcessor: AbstractProcessor() {
         filter = processingEnv.filer
         messager = processingEnv.messager
         composableTypeMirror = elementUtils
-            .getTypeElement(Class.forName(COMPOSABLE_CLASS_NAME).canonicalName)
+            .getTypeElement(COMPOSABLE_CLASS_NAME)
             .asType()
         textStyleTypeMirror = elementUtils
-            .getTypeElement(Class.forName(TYPE_STYLE_CLASS_NAME).canonicalName)
+            .getTypeElement(TYPE_STYLE_CLASS_NAME)
             .asType()
     }
 

